src/main/java/org/example/App.java

```diff
@@ -1,8 +1,18 @@
 package org.example;
 
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.Stream;
 
     public class App {
     public static void main( String[] args ) {
-        System.out.println( "Hello World!" );
+        System.out.println("Please enter your words with spaces between them:");
+        Scanner scanner = new Scanner(System.in);
+        String[] str = scanner.nextLine().split(" ");
```

Comment:
The task description says that list of elements should be consumed from arguments, e.g. java jar bla bla bla